src/prefect/server/api/flow_runs.py

```diff
@@ -95,6 +97,37 @@ async def update_flow_run(
     Updates a flow run.
     """
     async with db.session_context(begin_transaction=True) as session:
+        if PREFECT_EXPERIMENTAL_ENABLE_FLOW_RUN_INFRA_OVERRIDES.value():
```

Comment:
nit: You don't need to call `.value` here because all settings have `__bool__` defined to evaluate according to the truthiness of the value they contain.
```suggestion
        if PREFECT_EXPERIMENTAL_ENABLE_FLOW_RUN_INFRA_OVERRIDES:
```

Reply:
TIL!